scripts/run.sh

```diff
@@ -1,11 +1,26 @@
 #!/bin/sh
-if [ "$MYSQL_HOST" = "localhost" ] ||  [ "$MYSQL_HOST" = "127.0.0.1" ]; then
-	sh /app/install-mysql.sh
-fi
+
+set -e
+
+mysqld_safe &
+
+npm install npm@latest -g
+
+adduser -S cytube
+
+git clone -b 3.0 https://github.com/calzoneman/sync /home/cytube/app
+mkdir -p /home/cytube/certs
+cd /home/cytube/app
+sed -i 's/67c7c69a/ffdbce8/' package.json
+cp -f /scripts/config.docker.yaml /home/cytube/app
+chown -R cytube: /home/cytube
+
+su-exec cytube npm install
+su-exec cytube npm run build-server
```

Comment:
build-server is executed by postinstall when running npm install so this can be removed.

Reply:
Excellent, I'll update that later today.